src/components/browse/datasetlist/datasetheader/datasetdetails/datasetDetails.tsx

```diff
@@ -43,17 +48,22 @@ const DatasetDetails = (props: dataSetDetailsProps) => {
       const main_contact = dataAccessCommittee.main_contact;
       for (var item of dataAccessCommittee.has_member) {
         if (main_contact === null) {
-          return item.email === null ? mailId : item.email
+          mailId = (item.email === null || item.email === undefined) ? mailId : item.email
```

Comment:
Maybe we also need to check that it is a valid email too. Can't be too certain.

Reply:
This is a check performed by the backend. They have to give us a valid email ID.

Reply:
Doesn't pay to be extra cautious, but that's ok.